apps/web/app/modules/Auth/Register.page.tsx

```diff
@@ -1,91 +1,115 @@
-      <CardContent>
-        <form onSubmit={handleSubmit(onSubmit)} className="grid gap-4">
-          <div className="grid gap-2">
-            <Label htmlFor="firstName">{t("registerView.field.firstName")}</Label>
-            <Input id="firstName" type="text" placeholder="John" {...register("firstName")} />
-            {errors.firstName && (
-              <div className="text-sm text-red-500">
-                {t(errors.firstName.message ?? "registerView.validation.firstName")}
-              </div>
-            )}
-          </div>
-          <div className="grid gap-2">
-            <Label htmlFor="lastName">{t("registerView.field.lastName")}</Label>
-            <Input id="lastName" type="text" placeholder="Doe" {...register("lastName")} />
-            {errors.lastName && (
-              <div className="text-sm text-red-500">
-                {t(errors.lastName.message ?? "registerView.validation.lastName")}
-              </div>
-            )}
-          </div>
-          <div className="grid gap-2">
-            <Label htmlFor="email">{t("registerView.field.email")}</Label>
-            <Input id="email" type="email" placeholder="user@example.com" {...register("email")} />
-            {errors.email && (
-              <div className="text-sm text-red-500">
-                {t(errors.email.message ?? "registerView.validation.email")}
-              </div>
-            )}
-          </div>
-          <div className="grid gap-2">
-            <Label htmlFor="password">{t("registerView.field.password")}</Label>
-            <Input id="password" type="password" {...register("password")} />
-            {errors.password && (
-              <div className="text-sm text-red-500">
-                {t(errors.password.message ?? "registerView.validation.password")}
-              </div>
-            )}
+    <FormProvider {...methods}>
+      <Card className="mx-auto max-w-sm">
+        <CardHeader>
+          <CardTitle className="text-xl">{t("registerView.header")}</CardTitle>
+          <CardDescription>{t("registerView.subHeader")}</CardDescription>
+        </CardHeader>
+        <CardContent>
+          <form onSubmit={handleSubmit(onSubmit)} className="grid gap-4">
+            <div className="grid gap-2">
+              <Label htmlFor="firstName">{t("registerView.field.firstName")}</Label>
+              <Input id="firstName" type="text" placeholder="John" {...register("firstName")} />
+              {errors.firstName && (
+                <div className="text-sm text-red-500">
+                  {t(errors.firstName.message ?? "registerView.validation.firstName")}
+                </div>
+              )}
+            </div>
+
+            <div className="grid gap-2">
+              <Label htmlFor="lastName">{t("registerView.field.lastName")}</Label>
+              <Input id="lastName" type="text" placeholder="Doe" {...register("lastName")} />
+              {errors.lastName && (
+                <div className="text-sm text-red-500">
```

Comment:
Please notice that you repeat the same code, don't we have component for Errors? If not please create one.